2/homework/homework.ipynb

```diff
@@ -0,0 +1,2189 @@
+{
```

Comment:
To set seed in pytorch use this
```
    torch.backends.cudnn.deterministic = True
    torch.backends.cudnn.benchmark = False
    torch.manual_seed(seed)
```

Reply:
Prefer `.to(device)` instead of `cuda()` since this allows you to use cpu as well and limit your gpu usage

Reply:
It is really hard to review with large amounts of code in jupyter
Please explore https://ipython.readthedocs.io/en/stable/config/extensions/autoreload.html

Reply:
You do not have any evaluations metrics in your jupyter, how should I evaluate quality of your classifier?

Reply:
Thanks for the most up-to-date way of making PyTorch deterministic.

Reply:
**autoreload** is the feature I though is missing from jupyter. Thank you!

Reply:
You are right. Test loss is not very representative. Updated to include accuracy and AUC.